doc/examples/reconst_dki.py

```diff
@@ -54,6 +65,9 @@
                     & W_{xxzz} & W_{yyzz} & W_{xxyz} & W_{xyyz} & W_{xyzz}
                     & & )\\end{matrix}
 
+Example Tutotial
```

Comment:
Typo, but you can replace by `In Practice` or something similar. (avoid "example, tutorial)